tests/agents/test_pysmurf_controller_agent.py

```diff
@@ -155,10 +155,10 @@ def mock_set_current_mode(S, bgs, mode, const_current=True):
     return mock.MagicMock()
 
 
-def mock_biasstepanalysis(S, cfg, bgs, run_kwargs):
-    """mock_biasstepanalysis()
+def mock_biasstepanalysis():
+    """Mock Bias Step Analysis (bsa) object typically returned by sodetlib
+    operations.
 
```

Comment:
the `bgmap` object in the BiasStepAnalysis is an array of len `nchans` with the i-th element being the bias group that readout-channel `i` belongs to, with -1 being unassigned. To test this we should probably do something like `np.zeros(1000)` and check that the 0-th bg has 1000 channels assigned to it.